src/workspacer.Bar/Widgets/BatteryWidget.cs

```diff
@@ -11,8 +11,12 @@ namespace workspacer.Bar.Widgets
     public class BatteryWidget : BarWidgetBase
     {
         public Color LowChargeColor { get; set; } = Color.Red;
+        public Color MedChargeColor { get; set; } = Color.Yellow;
+        public Color HighChargeColor { get; set; } = Color.Green;
         public bool HasBatteryWarning { get; set; } = true;
         public double LowChargeThreshold { get; set; } = 0.10;
+        public double MedChargeThreshold { get; set; } = 0.50;
+        public double HighChargeThreshold { get; set; } = 1.00;
```

Comment:
Looking at the code below, I would argue `HighChargeThreshold` has no practical value and can be removed.